fluent-jdbc/src/main/java/org/codejargon/fluentjdbc/internal/mappers/DefaultObjectMapperRsExtractors.java

```diff
@@ -83,32 +83,42 @@ private static void basicTypes(Map<Class, ObjectMapperRsExtractor<?>> exs) {
         reg(exs, String.class, ResultSet::getString);
     }
 
-    private static void binaryTypes(Map<Class, ObjectMapperRsExtractor<?>> exs) {
+    private static void binaryTypes(Map<Class, ObjectMapperRsExtractor<?>> exs, boolean preferDirect) {
         reg(exs, byte[].class, (rs, i) -> {
             Blob blob = rs.getBlob(i);
             if (blob == null) {
                 return null;
             }
             byte[] data = blob.getBytes(0, (int) blob.length());
-            blob.free();
+            try {
+              blob.free();
+            } catch (Exception e) {
```

Comment:
This may throw SQLException other than non supported exception. it's not good to ignore it completely. fixed on feature branch.